build.gradle

```diff
@@ -91,10 +91,10 @@ ext.library = [
     google_api_services_storage: "com.google.apis:google-api-services-storage:v1-rev71-$google_clients_version",
     google_auth_library_credentials: "com.google.auth:google-auth-library-credentials:$google_auth_version",
     google_auth_library_oauth2_http: "com.google.auth:google-auth-library-oauth2-http:$google_auth_version",
-    google_cloud_core: "com.google.cloud:google-cloud-core:1.0.2",
-    google_cloud_core_grpc: "com.google.cloud:google-cloud-core-grpc:$grpc_version",
+    google_cloud_core: "com.google.cloud:google-cloud-core:1.16.0",
+    google_cloud_core_grpc: "com.google.cloud:google-cloud-core-grpc:1.16.0",
```

Comment:
I do not think beam needs to directly depend on this as well.

Reply:
I think it does. I can't remember exactly where. If you remove these deps from pom, the build fails. BEAM verifies that dependency is declared if and only if code uses something from that dependency.